### ISSUE TITLE
inline

### ISSUE THREAD

**Issue by mohamedyounis30002000** [aldocano29/AmandaDocxPrinter#13]
You did a nice job

plz let me know how to show the output inline

**Comment by imperialgas2020** [comment]
yes i need to show the file in the form inline instead of downloaded it
and it will be fantastic if i can add watermark
